lib/resources/example_json/r4_examples_slot.rb

```diff
@@ -0,0 +1,232 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+
+    R4_SLOT_ENTRY ||= {
+      'resourceType': 'Slot',
+      'id': '454030723-39997807-50069221-0',
+      'meta': {
+        'versionId': '3423957v2380283',
+        'lastUpdated': '2020-05-08T18:13:55Z'
+      },
+      'text': {
+        'status': 'generated',
+        'div': '<div xmlns="http://www.w3.org/1999/xhtml"><p><b>Slot</b></p>'\
+        '<p><b>Service Type</b>: SK DOT apt 2</p><p><b>Start</b>: May  8, 2020  2:00 P.M. UTC</p>'\
+        '<p><b>End</b>: May  8, 2020  3:00 P.M. UTC</p><p><b>Schedule Reference</b>'\
+        ': 454030723-39997807-50069221-0</p><p><b>Status</b>: Free</p></div>'
+      },
+      'extension': [
+        {
+          'valueReference': {
+            'reference': 'Location/39997807'
+          },
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/scheduling-location'
+        }
+      ],
+      'serviceType': [
+        {
+          'coding': [
+            {
+              'system': 'https://fhir.cerner.com/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/codeSet/14249',
+              'code': '454030723',
+              'display': 'SK DOT apt 2',
+              'userSelected': true
+            }
+          ],
+          'text': 'SK DOT apt 2'
+        }
+      ],
+      'schedule': {
+        'reference': 'Schedule/454030723-39997807-50069221-0'
+      },
+      'status': 'free',
+      'start': '2020-05-08T14:00:00Z',
+      'end': '2020-05-08T15:00:00Z'
+    }.freeze
+
+    R4_SLOT_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': 'bf750997-bfdd-4ac3-bcc2-34776d05b46e',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-open.devcerner.com/r4/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/Slot?service-type=https%3A%2F%2Ffhir.cerner.com%2Feb2384f8-839e-4c6e-8b29-18e71db1a0b1%2FcodeSet%2F14249%7C517975329&start=ge2020-07-22T20%3A04%3A19Z&start=lt2020-11-08T18%3A13%3A55Z&_count=50&schedule.actor=Practitioner%2F2911963'
```

Comment:
Use https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/. Same comment for all below URLs

Reply:
updated